wagtail/contrib/simple_translation/models.py

```diff
@@ -9,8 +13,28 @@ class SimpleTranslation(Model):
     - `remove_stale_contenttypes`, will drop the perm
     - `dump_data`, will complain about the missing table
     """
+
     class Meta:
         default_permissions = []
         permissions = [
-            ('submit_translation', "Can submit translations"),
+            ("submit_translation", "Can submit translations"),
         ]
+
+
+@hooks.register("after_create_page")
+def after_create_page(request, page):
+    """Creates page aliases in other locales when a page is created.
+
+    Whenever a page is created under a specific locale, this signal handler
+    creates an alias page for that page under the other locales.
+
+    e.g. When an editor creates the page "blog/my-blog-post" under the English
+    tree, this signal handler creates an alias of that page called
+    "blog/my-blog-post" under the other locales' trees.
+    """
+    if getattr(settings, "WAGTAILSIMPLETRANSLATION_SYNC_PAGE_TREE", False):
+        # Check if the source tree needs to be synchronised into any other trees
+        # Create aliases in all those locales
+        for locale in Locale.objects.exclude(pk=page.locale_id):
```

Comment:
Note: this may be another place to add to https://github.com/wagtail/rfcs/pull/72 when the site has lots of locales